### PR TITLE
Fix requirements

### PR DESCRIPTION
**Pull request by sharkykh**
- [x] Update requirements
- [x] ~Decode~ Encode data to bytes if needed
		**Remove `builtins` import,** as on Python 2 it comes from the `future` package.

@medariox / @p0psicles

### CODE REVIEW

**Review by p0psicles — APPROVED** [review]

File: adba/aniDBlink.py [hunk 1]
```diff
@@ -213,7 +212,13 @@ def _send(self, command):
         command.started = time()
         data = command.raw_data()
 
-        self.sock.sendto(bytes(data, "ASCII"), self.target)
+        # Decode data to bytes if needed
```
Comment (medariox) [inline adba/aniDBlink.py:215]: `Encode` data to bytes?

Comment (sharkykh) [reply adba/aniDBlink.py:215]: Sorry, yes. Of course.

Comment (sharkykh) [reply adba/aniDBlink.py:215]: Can you just edit it on master?

Comment (medariox) [reply adba/aniDBlink.py:215]: Done 👌

**Pull request merged**